src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -141,6 +146,7 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
           ...attributes,
         }}
         className={noTranslate ? 'no-translate' : undefined}
+        dir={dir}
```

Comment:
No strong opinion here but it seems like we know password is always LTR so we could consider hardcoding it in here instead of allowing it to be configurable

Reply:
actually this InputPassword is also used for the security question answer, which needs to inherit RTL